lib/timeline-reporter.js

```diff
@@ -157,13 +137,13 @@ class TimelineReporter extends EventEmitter {
                 context,
                 browser: runnerInfo.sanitizedCapabilities,
                 duration: humanizeDuration(_duration, round),
-                embedScreenshots: embed,
+                embedScreenshots: embedImages,
```

Comment:
If the option coming back `this.getReporterOptions()` would be called `embedScreenshots`, this could just be `embedScreenshots`

Reply:
Might be unnecessary work at this point for this PR, though

Reply:
will update I thought of changing it too